core/arch/arm/kernel/thread_optee_smc.c

```diff
@@ -32,7 +32,7 @@ void thread_handle_fast_smc(struct thread_smc_args *args)
 	thread_check_canaries();
 
 #ifdef CFG_VIRTUALIZATION
-	if (!virt_set_guest(args->a7)) {
+	if (virt_set_guest(guest_id)) {
```

Comment:
Looks like some rebase artifact or copy-paste mistake. There is no `guest_id` variable introduced in this commit